src/SFA.DAS.Commitments.Infrastructure.UnitTests/TestHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using KellermanSoftware.CompareNetObjects;
+using Newtonsoft.Json;
+
+namespace SFA.DAS.Commitments.Infrastructure.UnitTests
+{
+    public static class TestHelper
+    {
+        public static T Clone<T>(T source)
+        {
+            var serialized = JsonConvert.SerializeObject(source);
+            return JsonConvert.DeserializeObject<T>(serialized);
+        }
+
+        public static bool EnumerablesAreEqual<T>(IEnumerable<T> expected, IEnumerable<T> actual)
```

Comment:
Not used but looks very useful

Reply:
It's used now (after addressing your last comment) 😁